src/app/components/AccountMenu/index.tsx

```diff
@@ -52,10 +52,10 @@ function AccountMenu({ showOptions = true }: Props) {
   async function selectAccount(accountId: string) {
     setLoading(true);
     try {
-      setAccountId(accountId);
       await msg.request("selectAccount", {
         id: accountId,
       });
+      setAccountId(accountId);
```

Comment:
Why did you change the order here? 

This leads to some side effects (e.g. the avatar changes only after the account has been loaded) :thinking:

Reply:
is that a problem? 

I wanted to avoid some react dependency to do requests before the new account was switched.